server/core/src/lib.rs

```diff
@@ -748,7 +748,7 @@ impl Display for TaskName {
 
 pub struct CoreHandle {
     clean_shutdown: bool,
-    tx: broadcast::Sender<CoreAction>,
+    pub tx: broadcast::Sender<CoreAction>,
```

Comment:
Why does this need to be pub?

Reply:
so we can access it in the tokio::select loop

Reply:
pub(crate) then maybe?

Reply:
Nope, won't work - CoreHandle is over in server/core, while it's being used in server/daemon - different crates 😱